qemu/tests/cdrom.py

```diff
@@ -359,21 +359,34 @@ def get_match_cdrom(session, serial_num):
         :return match_cdrom: the cdrom in guest which is corresponding
                              with the CML according to the serial number.
         """
+
+        def get_cdrom_by_serial(session, serial_num):
+            """
+            Get match cdrom from disk list, if not found return None.
+
+            :param session: VM session
+            :param serial_num: cdrom serial number
+            """
+            show_serial_num = "ls -l /dev/disk/by-id"
+            serial_num_output = session.cmd_output(show_serial_num)
+            if serial_num_output:
+                for line in serial_num_output.splitlines():
+                    if utils_misc.find_substring(str(line), str(serial_num)):
+                        serial_cdrom = line.split(" ")[-1].split("/")[-1]
+                        return serial_cdrom
```

Comment:
For this func(), can use the https://github.com/avocado-framework/avocado-vt/pull/324 directly now.